doc/org/build/networking.md

```diff
@@ -0,0 +1,256 @@
+
+Clients send messages at fixed intervals in the `networkSendSystem`. Messages are sent at fixed intervals (rather than anytime entity state is updated) so that frequently updated components (like `NetworkedTransform`) do not cause a client to flood the network with an unnecessary amount of `UpdateMessage` s.
+
+Clients receive messages outside of frame boundaries, and simply queue them for processing. Clients process queued messages each frame in the `networkReceiveSystem`.
+
+
+<a id="orgabca6d0"></a>
+
+# The `Networked` Component
+
+Networked entities are any entities with the `Networked` component. The `Networked` component contains:
+
+-   A (networked) `id`, which clients use to uniquely identify this entity across the network. This cannot simply be the `eid` of an entity, because `eid` s are assigned locally to each client.
+-   A `creator`, which is used at various times to determine whether or not an entity should be created or removed.
+-   An `owner`, which is used to determine which `client` has authority to update the state of this entity.
+-   A `lastOwnerTime`, which is used to determine the most recent time that an `owner` was assigned.
+-   A `timestamp`, which is the most recent time the networked state of this entity has been updated.
+
+The `creator` can be set to a `ClientId`, `"reticulum"` , or a `NetworkID` .
+
+-   When the `creator` is a `ClientID`, it means that a client in the room has caused this &ldquo;root&rdquo; entity (and its descendants) to be created, and the entity (and its descendants) should be removed when the client leaves the room.
+-   When the `creator` is `"reticulum"` , it means that the Reticulum is responsible for deciding whether this &ldquo;root&rdquo; entity should be created or removed.
+-   When the `creator` is a `NetworkID`, it means that the entity is a descendant of a &ldquo;root&rdquo; entity, and its creation/removal will be subject to its ancestor.
+
+Conceptually, the `creator` and the `owner` act as authorities over two facets of a networked entity.
+
+-   The `creator` is the authority over the entity&rsquo;s existence. Thus, it is checked when processing `CreateMessage` s and before an entity may be removed. For an entity to be created, its `creator` must have the appropriate permission. An entity&rsquo;s `creator` changes infrequently. It only happens when a client `pins` (or `unpins`) an entity, which changes the `creator` to (or from) `"reticulum"`.
+-   The `owner` is the authority over the entity&rsquo;s current state. Thus, it is associated with `UpdateMessage` s. The `owner` is expected to change regularly, whenever a new client performs an action on an entity. The `takeOwnership` and `takeSoftOwnership` functions allow a client to establish itself as the `owner` of an entity.
```

Comment:
Owner concept may be a bit complex to understand so would be better to make a separated section for it.